modules/ml/include/opencv2/ml.hpp

```diff
@@ -1164,6 +1164,14 @@ class CV_EXPORTS_W RTrees : public DTrees
      */
     CV_WRAP virtual Mat getVarImportance() const = 0;
 
+    /** Returns the result of each individual tree in the forest.
+    In case the model is a regression problem, the method will return each of the trees'
+    results for each of the sample cases. If the model is a classifier, it will return 
+    a Mat with samples + 1 rows, where the first row gives the class number and the 
+    following rows return the votes each class had for each sample.
+    */
+    CV_WRAP void getVotes(InputArray, OutputArray, int) const;
```

Comment:
I missed this: all parameters should have names and be documented. Meaning of the last integer parameter is unclear without looking through the code.